mysite/_config.php

```diff
@@ -1 +1,11 @@
 <?php
+
+use SilverStripe\Security\PasswordValidator;
+use SilverStripe\Security\Member;
+
+// remove PasswordValidator for SilverStripe 5.0
+$validator = new PasswordValidator();
+
+$validator->minLength(8);
+$validator->checkHistoricalPasswords(6);
+Member::set_password_validator($validator);
```

Comment:
If we add this into a call_user_func callback it'll avoid creating global variables (as far as I know)

Reply:
global variables has been fixed in an earlier pull request by @kinglozzer :)
so it's not longer an issue for this case

Reply:
Here's a link: https://github.com/silverstripe/silverstripe-framework/pull/7493